airdrop/infrastructure/models.py

```diff
@@ -121,6 +121,11 @@ class UserRegistration(Base, AuditClass):
 
 class UserReward(Base, AuditClass):
     __tablename__ = "user_rewards"
+    airdrop_id = Column(
+        BIGINT,
+        ForeignKey("airdrop.row_id", ondelete="RESTRICT"),
```

Comment:
guess we have to use `CASCADE` for ondelete. 
Please confirm with @prashantramangupta

Reply:
Interesting. 
Could you attach any related blog or docs regt the same in this PR description? 
Will be helpful for future reference.

Reply:
https://medium.com/@dilongfa/understand-about-on-update-and-on-delete-5d465c538501